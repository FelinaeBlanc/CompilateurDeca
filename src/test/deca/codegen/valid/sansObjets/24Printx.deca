{
    float a = 2.25; 
    float b = 1.5; 

    printx(a + b);
    println(); 
    printx(a - b); 
    println(); 
    printx(a * b); 
    println(); 
    printx(a / b); 
    println(); 
    printx(-b); 
}