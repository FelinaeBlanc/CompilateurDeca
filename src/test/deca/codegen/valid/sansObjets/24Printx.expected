0x1.ep+1
0x1.8p-1
0x1.bp+1
0x1.8p+0
-0x1.8p+0
